### PR TITLE
FIX: Add neat workspace settings

### PR DESCRIPTION
**Pull request by cristian-aldea**
Added workspace settings to attempt to make some of our workflow a bit more consistent. These settings are applied to everyone who works on our repository with vscode can be updated and added upon over time.

A few examples of vscode settings I felt we should all have:
1. defaultFormatter: 100000 times I've seen one of our teammates complain that their formatters isn't working, when in fact they had multiple formatters present for a given file type and didn't pick a default one.
2. codeActionsOnSave: Our imports are all over the place, with no consistency as to how they are ordered, the organizeImports on save fixes this, and uses our linter, eslint, with no need for extra plugins! Another bonus, it cleans up unused imports!
3. formatOnSave. Self-explanatory, I think everyone should have this.

I'm open to suggestions of other things we'd need to add for these workspace settings.

### CODE REVIEW

**Review by leojrsilao — DISMISSED** [review]
LGTM! It will definitely be beneficial for people who have multiple formatter plugins and do not have the right formatter set up for the project. The code will definitely be cleaner with organized imports. Good fix!!🚀 🚀

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": true
```
Comment (r-saba) [inline .vscode/settings.json:3]: Thanks, this was driving my insane! 
Should we run this on all files in this PR? Other PR's editing existing files will have a lot of changes unrelated to the PR

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  "editor.formatOnSave": true,
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[json]": {
```
Comment (r-saba) [inline .vscode/settings.json:9]: Just a question where can I find the documentation for all these setting options?

Comment (cristian-aldea) [reply .vscode/settings.json:9]: Look into stuff related to `setting.json` for VSCode. This is a good page: https://code.visualstudio.com/docs/getstarted/settings

File: .vscode/settings.json (hunk 1 above)
Comment (cristian-aldea) [reply .vscode/settings.json:3]: Sure I'll get on that

Comment (r-saba) [reply .vscode/settings.json:3]: Great thanks! Anyone who rebases will have simple merge conflicts but it shouldn't be an issue. I tested rebasing on this branch locally it was straightforward.

**Review by r-saba — APPROVED** [review]
LGTM! Good stuff, thanks

**Review by bit172 — APPROVED** [review]
LGTM!

**Review by leojrsilao — APPROVED** [review]
Just have to fix the merge conflicts and LGTM!

**Comment by cristian-aldea** [comment]
Rebased and ran tests, linting, e2e and served the apps. Everything still seems good, so i'll be merging.

**Pull request merged**